lithium/lithium.py

```diff
@@ -55,7 +55,8 @@ def readTestcase(self, filename):
             # Determine whether the f has a DDBEGIN..DDEND section.
             for line in f:
                 if line.find(b"DDEND") != -1:
-                    raise LithiumError("The testcase (%s) has a line containing 'DDEND' without a line containing 'DDBEGIN' before it." % self.filename)
+                    raise LithiumError("The testcase ({}) has a line containing 'DDEND' ".format(self.filename) +
+                                       "without a line containing 'DDBEGIN' before it.")
```

Comment:
> Also, I may have been under the impression that the % modulo operator was on its way out via PEP 3101.

Ditto in the [Python 2 docs](https://docs.python.org/2/library/stdtypes.html#str.format) for `str.format`:

`
This method of string formatting is the new standard in Python 3, and should be preferred to the % formatting described in String Formatting Operations in new code.
`